README.md

```diff
@@ -1,6 +1,52 @@
 # Library for Linear Reference System
 
-TODO: explain what this library is for
+[Linear Reference](https://en.wikipedia.org/wiki/Linear_referencing) allows to identify a position along a curve (a road, a canal, a railroad…) relative to fixed reference points. 
+
+We call:
+- **anchor** a reference point: it can be milestones, a landmark, an intersection…
+- **scale** the list of _anchors_ and the theoretical distance between them,
+- **curve** the physical description of the object to reference,
+- **linear referencing method** (LRM) the combination of a _scale_ and a _curve_,
+- **linear referencing method** (LRS) the complete set of data.
```

Comment:
**linear referencing system** (LRS) the complete set of data.

Reply:
good catch! thank you